Sources/OnboardingFeature/OnboardingStepView.swift

```diff
@@ -58,42 +58,39 @@ struct OnboardingStepView: View {
             Group {
               if self.viewStore.step == .step1_Welcome {
                 FullscreenStepView(
-                  Text("Hello!\nWelcome\nto ")
+                  Text("Hello!\nWelcome to ")
                     + Text("isowords").fontWeight(.medium)
-                    + Text(", a\nword game.")
+                    + Text(", a word game.")
                 )
               }
               if self.viewStore.step == .step2_FindWordsOnCube {
                 FullscreenStepView(
-                  Text("The point of\n")
-                    + Text("the game is\n")
-                    + Text("to find words\n")
-                    + Text("on a ") + Text("cube").fontWeight(.medium) + Text(".")
+                  Text("The point of the game is to find words on a ")
+                    + Text("cube").fontWeight(.medium)
+                    + Text(".")
                 )
               }
               if self.viewStore.step == .step3_ConnectLettersTouching {
                 FullscreenStepView(
-                  Text("Words are\n")
-                    + Text("formed by\n")
-                    + Text("connecting\n")
-                    + Text("letters that are\n")
-                    + Text("touching").fontWeight(.medium) + Text(".")
+                  Text("Words are formed by connecting letters that are\n")
```

Comment:
should we drop this new line too?

```suggestion
                  Text("Words are formed by connecting letters that are")
```